### ISSUE TITLE
Long process names can lead to tables overflowing in 2-table layout

### ISSUE THREAD

**Issue by cmutel** [cauldron/bw_matchbox#6]
@lilliputten Has done some work in https://github.com/lilliputten/bw_matchbox/tree/dev-230729-two-tables-layout-bug already.

```Implemented 'smth in the middle' solution (for tables layout):

- All columns, except for the name, are fixed in width.
- Text inside the cells clipping with ellipsises, if possible.
- Adjusted (reduced, where it was possible) table paddings.
```

**Comment by cmutel** [comment]
> I think it is also possible to reduce the width of the space between the two tables (didn't touched it at all -- this is still the skeleton's area of responsibility). (It seems to be too wide now.)

I agree, let's create a custom copy of the table and reduce some of this unnecessary white space.

**Comment by lilliputten** [comment]
This has already been done in the latest fixes. The space between tables was reduced to 15px.

**Comment by cmutel** [comment]
If the PR isn't merged it isn't fixed :) But nice to know that it is on the way!

**Comment by cmutel** [comment]
Fixed in #11
